File: Coiner/ClientApp/src/app/models/blockChainOfferDto.ts
import { OfferTypeEnum } from "./enums/offerType";

export class BlockChainOfferDto {
    userId: number;
    privateKey: string;
    productName: string;
    currency: string;
    unitPrice: number;
    currencyQuantity: number;
    productQuantity: number;
    offerType: OfferTypeEnum;
    totalPrice?: number;
    commissionFees: number = 0;
}

File: Coiner/Controllers/ModelsDto/BlockChainOfferDto.cs
using Coiner.Business.Helpers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Coiner.Controllers.ModelsDto
{
    public class BlockChainOfferDto
    {
        public int UserId { get; set; }

        public string PrivateKey { get; set; }

        public string ProductName { get; set; }

        public string Currency { get; set; }

        public decimal CurrencyQuantity { get; set; }

        public decimal ProductQuantity { get; set; }

        public OfferTypeEnum OfferType { get; set; }

        //commissionFees
        public decimal CommissionFees { get; set; } 

    }
}
